packages/components/src/color-picker/style.scss

```diff
@@ -27,7 +27,7 @@
 
 .components-color-picker {
 	width: 100%;
-	overflow: hidden;
+	overflow: visible;
```

Comment:
Im not finding this change necessary to fix the bug.  However, the change to `visible` in `.components-color-picker__saturation-color` does seem to be what does it!

Did you have any other reasons for changing the rule on this selector as well?

Reply:
It does seem like I can remove that. So I did. Thanks again.

I rebased in hopes that the checks will now pass. 🤞